### DISCUSSION TITLE
Custom text isn't syncing across browsers/machines

### DISCUSSION THREAD

**Discussion by kiug** [monkeytypegame/monkeytype#4740] [Questions]
Why custom text (and progress) isn't syncing across different browsers/machines?

**Comment by Miodec** [comment]
Its not meant to. Custom text is only saved locally

**Comment by karl** [comment]
Would you be interested in a PR to enabling syncing of custom text across machines?

This would be really useful to allow my daughter to practice the same custom word list at school and at home.
